src/api.js

```diff
@@ -72,6 +72,17 @@ export const loginUser = async user => {
     throw error;
   }
 };
+
+export const getUserById = async id => {
+  try {
```

Comment:
If you're only throwing the error, you should be able to just remove the try catch all together and just do the logic inside the try

Reply:
We then get a "missing a catch or finally clause" error?

Reply:
Updated my comment sorry